FWCore/Integration/test/AcquireIntStreamFilter.cc

```diff
@@ -33,7 +33,7 @@ namespace edmtest {
   };
 
   AcquireIntStreamFilter::AcquireIntStreamFilter(edm::ParameterSet const& pset)
-      : m_token{edm::Service<test_acquire::WaitingService>{}->getToken()} {
+      : m_token{edm::Service<test_acquire::WaitingService>{} -> getToken()} {
```

Comment:
This looks worse too (although only slightly compared to some of the others)

Reply:
looks like a bug to me, we use google formating style and it [a] says 
```
No spaces around period or arrow. Pointer operators do not have trailing spaces.
```

and strangely clang-format only added spaces in this line while in the same file there are other instances of `->` where it did not suggest to add the spaces. May be it is becase of `{}` before the arrow. 
 
[a]
https://google.github.io/styleguide/cppguide.html#Pointer_and_Reference_Expressions

Reply:
> May be it is becase of `{}` before the arrow.

That hypothesis would be simple to test with
```suggestion
      : m_token{edm::Service<test_acquire::WaitingService>()->getToken()} {
```

Reply:
yes, if we have `()->` then clang-format does not add extra spaces but for `{}->` it does. Should I apply your change @makortel ?

Reply:
Let's go for `()->` then.